src/main/resources/db/migration/V1__init_table.sql

```diff
@@ -6,3 +6,6 @@ CREATE TABLE FEATURES (
   IMAGE_URL VARCHAR(255) not null,
   PRIMARY KEY (ID)
 );
+
+INSERT INTO FEATURES(NAME, DESCRIPTION, AUTHOR, IMAGE_URL)
```

Comment:
To do twojej migracji daj albo do osobnej nie mozesz zmieniac starych migracji, bo wprowadzisz niespojnosc.